openassessment/xblock/static/js/spec/lms/oa_response.js

```diff
@@ -892,10 +892,12 @@ describe("OpenAssessment.ResponseView", function() {
         $(firstDescriptionField1).val('test1');
         $(firstDescriptionField2).val('test2');
 
-        // user finishes adding descriptions which enables submit button
+        // user finishes adding descriptions which enables the upload button
         view.checkSubmissionAbility(true);
-        expect(view.submitEnabled()).toBe(true);
         expect(view.uploadEnabled()).toBe(true);
+
+        // the submit button remains disabled until files are uploaded
+        expect(view.submitEnabled()).toBe(false);
```

Comment:
Should there be a test for when files are uploaded?

Reply:
A "re-enable submit button after upload" step?

Reply:
I'd argue that is already covered by existing tests. See "tests that new file uploads are appended". These changes are only concerned with Submit/Upload button states while staging.